### PR TITLE
Add CIFAR10-H

### PR DESCRIPTION
**Pull request by gortizji**
# Add Dataset

* Dataset Name: CIFAR10-H
* Issue Reference: https://github.com/tensorflow/datasets/issues/4603

## Description
A re-labeled version of CIFAR-10's test set with soft-labels coming from real human annotators. For every pair (image, label) in the original CIFAR-10 test set, it provides several additional labels given by real human annotators as well as the average soft-label. The training set is identical to the one of the original dataset.

## Checklist

* [X] Address all TODO's
* [X] Add alphabetized import to subdirectory's `__init__.py`
* [X] Run `download_and_prepare` successfully
* [X] Add [checksums file](https://www.tensorflow.org/datasets/add_dataset#2_run_download_and_prepare_locally)
* [X] Properly cite in `BibTeX` format
* [X] Add passing test(s)
* [X] Add test data
* [ ] ~If using additional dependencies (e.g. `scipy`), use [lazy_imports](https://www.tensorflow.org/datasets/add_dataset#extra_dependencies) (if applicable)~
* [ ] ~Add data generation script (if applicable)~
* [X] [Lint](https://www.tensorflow.org/datasets/add_dataset#5_check_your_code_style) code

### CODE REVIEW

**Comment by pierrot0** [comment]
Thank you for your contribution!
And sorry for the delay here.
